src/components/v2/Layout/Header/ConnectButton.tsx

```diff
@@ -0,0 +1,30 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+
+import { AuthContext } from 'context/AuthContext';
+import { truncateAddress } from 'utilities/truncateAddress';
+import { SecondaryButton, IButtonProps } from '../../Button';
+
+export interface IConnectButton extends IButtonProps {
+  openAuthModal: () => void;
```

Comment:
Naming this prop `onClick` is more consistent and opens the possibility for this component to have a different interaction besides opening a modal

Reply:
You're right 👍